lib/urlwatch/tests/test_handler.py

```diff
@@ -99,8 +99,8 @@ def py_files():
 
 
 class ConfigForTest(CommandConfig):
-    def __init__(self, config, urls, cache, hooks, verbose):
-        super().__init__([], 'urlwatch', os.path.dirname(__file__), root, config, urls, hooks, cache, verbose)
+    def __init__(self, config, urls, cache, hooks, verbose, args=[]):
```

Comment:
`args=[]` is a [mutable default argument](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments).

Something immutable (e.g. `args=frozenset()` or the empty tuple `args=()`) would be totally acceptable, though.

Reply:
Every day is a school day, thanks